test/acceptance/user.controller.acceptance.ts

```diff
@@ -130,6 +133,74 @@ describe('UserController', () => {
     });
   });
 
+  describe('authentication functions', () => {
+    it('login returns a valid token', async () => {
+      const newUser = await userRepo.create(user);
+      await client
+        .post('/users/login')
+        .send({email: newUser.email, password: newUser.password})
+        .expect(200)
+        .then(getToken);
+
+      function getToken(res: Response) {
+        // is it good to return the token as text?
```

Comment:
Not really. Please return a data object serialized as JSON, pass the token in a property of that object. For example:

```json
{
  "token": "header.payload.signature"
}
```